cmd/maya-apiserver/cstor-operator/spc/handler_test.go

```diff
@@ -372,10 +402,12 @@ func TestCurrentPoolCount(t *testing.T) {
 func TestPendingPoolCount(t *testing.T) {
 	fakeKubeClient := fake.NewSimpleClientset()
 	fakeOpenebsClient := openebsFakeClientset.NewSimpleClientset()
+	fakeNDMClient := ndmFakeClientset.NewSimpleClientset()
```

Comment:
You need to move out this code from commented section

Reply:
Updated the code